AWS/Foundational/note.txt

```diff
@@ -0,0 +1 @@
+These are all optional courses, not sure if anyones even gonna go for these?
```

Comment:
Please try to refrain from asking questions in documentation.

Reply:
I deleted that already